docs/windows_developer_guide.md

```diff
@@ -27,19 +27,38 @@ Install build tools for VS:
 
 https://aka.ms/vs/17/release/vs_BuildTools.exe
 
-Mark c++ Desktop app and v142 CPP platform toolset.
+Mark required options for installation:
+- c++ Desktop development with C++
+- MSVC v143 CPP - VS 2022 C++ platform toolset.
+- C++ CMake tools for Windows platform toolset.
+- MSVC v142 CPP - VS 2022 C++ platform toolset.
```

Comment:
Do we need both toolsets?

Reply:
Yes, the 143 is required to install cmake.

Reply:
Why do we keep information about v142 then?

Reply:
Because we need cmake - required 143, and we need v142 to compile OVMS.

Reply:
Will it be possible to unify in the future? Because it sounds like asking for problems in the future 😕